### ISSUE TITLE
[tate] [CJK] Unusable in master because よっ runs together

### ISSUE THREAD

**Issue by ctrlcctrlv** [sile-typesetter/sile#1452]
![image](https://user-images.githubusercontent.com/838783/175027066-07d11ae4-cf7b-4165-a059-712a7e90e710.png)

**Comment by alerque** [comment]
Can you include some MWE code of the shortest document string you see failing? I'm chasing gremlins...

**Comment by ctrlcctrlv** [comment]
Sorry for late reply, I finished what I wanted this for in upLaTeX so was distracted :blush: 

![image](https://user-images.githubusercontent.com/838783/175122291-e1cc6b75-89d5-420b-b740-a0dd85c70608.png)
![image](https://user-images.githubusercontent.com/838783/175122311-be8c598f-4785-4968-9da0-0a5819ed8de6.png)
![image](https://user-images.githubusercontent.com/838783/175122339-76babd0c-16ce-4775-9c36-8eb0c0c5aacd.png)

It'd be great if I could do something like that in SILE but SILE is still has way too glitchy CJK stack.

**Comment by ctrlcctrlv** [comment]
Wait, no, I was testing with the non-master #1447.

I just pulled your master and got:

![image](https://user-images.githubusercontent.com/838783/175123107-bcfff6d2-a5a3-4b5e-9a5f-d1b42590c2b6.png)

Now I'll try w/#1451.

**Comment by ctrlcctrlv** [comment]
Sorry @alerque #1451 makes no difference to this bug.

![image](https://user-images.githubusercontent.com/838783/175123477-0f7dda01-9789-47a6-bd47-df81b820af28.png)

**Comment by alerque** [comment]
Pardon my ignorance here, but what is this MWE *supposed* to look like? Is that character in it's own column supposed to be hanging out inline with the others?

I just spun up some old versions via Docker and I don't see anything different in this document rendered in v0.11.1, v0.12.0, v0.12.5, or v0.13.1.

**Comment by ctrlcctrlv** [comment]
If that's the case it's always been broken. Tate has many issues so I haven't used it in my projects extensively.

![image](https://user-images.githubusercontent.com/838783/175130475-550123d6-f479-4d81-b3a7-0fe65a4dbc43.png)

**Comment by Omikhleia** [comment]
FWIW, @ctrlcctrlv 's MWE

> ```
> \begin[papersize=a4,class=jplain,layout=tate]{document}
> \set[parameter=font.family, value=Noto Serif CJK JP]
> \set[parameter=font.features, value=vrt2]%doesn't help
> 
> 他人によって
> \end{document}
> ```

On SILE 0.12.5 / Lua 5.2 / Ubuntu 20.04

![image](https://user-images.githubusercontent.com/18075640/175426471-145b5c86-25a8-42da-99b5-b9effe72ee0c.png)

So I am not sure which bug you are chasing...

The font info, in case it helps...
![image](https://user-images.githubusercontent.com/18075640/175425284-827a061c-e9be-4cb2-8b3e-d9a19190fce0.png)

Oh... Wait. It's an old version already... Let's fetch a newer one.

![image](https://user-images.githubusercontent.com/18075640/175426604-1119f471-fb9c-4192-abae-8ee36eef4513.png)

![image](https://user-images.githubusercontent.com/18075640/175426711-a716c5a1-7894-455a-974c-b2ac3205f8dd.png)

Ooops?

**Comment by alerque** [comment]
Paging @simoncozens, Simon Cozens, cleanup on isle 3. :stuck_out_tongue: Maybe you can at least tell us if this is a font bug, a misuse of harfbuzz, or SILE being bad.

**Comment by Omikhleia** [comment]
Oh but wait again...
```
\begin[papersize=a4, class=jplain, layout=tate]{document}
\set[parameter=font.family, value=Noto Serif CJK JP]
\set[parameter=font.features, value=vrt2]%doesn't help
\language[main=ja] --- HERE HERE HERE

他人によって
\end{document}
```
And suddenly the issue disappears even with that 2.001 font. Without it, it seems `SILE.nodeMakers.ja` is unused...

**Comment by ctrlcctrlv** [comment]
Ugh thanks it's maybe because I removed the language declaration from `tate` as badly scoped (although 縦 【たて, _tate_】 is a Japanese word, and Simon, _much like me_, is most familiar with Japanese typesetting, `tate` is equally applicable to Ainu, Okinawan, Chinese, Korean, etc.

**Comment by ctrlcctrlv** [comment]
(The language declaration shouldn't go back in `tate`, it should go in `jplain`.)

**Comment by ctrlcctrlv** [comment]
I have a patch

**Comment by alerque** [comment]
Closed via #1452.
